labellab-client/src/components/project/images.js

```diff
@@ -1,26 +1,14 @@
-import ReactDOM from 'react-dom'
-import React, { Component } from 'react'
-import { connect } from 'react-redux'
+import React, {Component} from 'react'
```

Comment:
Lots of unnecessary changes here. Only differences in white space or indentation. Please revert them.